src/app/[locale]/requests/components/Requests.tsx

```diff
@@ -167,6 +171,30 @@ function Requests() {
                 setOpenModerationRequestCount(openMRCount)
                 setClosedModerationRequestCount(closedMRCount)
             })
+        void fetchData('clearingrequests')
+            .then((clearingRequests: EmbeddedClearingRequest) => {
+                let openCRCount = 0
+                let closedCRCount = 0
+                clearingRequests['_embedded']['sw360:clearingRequests']
+                .filter((item: ClearingRequest) => {
+                    if (  item.clearingState === 'NEW' ||
```

Comment:
```suggestion
                    if ( item.clearingState === 'NEW' ||
```
Remove redundant space